mern-ecommerce/controllers/user.js

```diff
@@ -40,6 +41,32 @@ exports.update = (req, res) => {
   );
 };
 
+exports.create = (req, res) => {
+  const category = new Category(req.body);
+  category.save((err, data) => {
+    if (err) {
+      return res.status(400).json({
+        error: errorHandler(err),
+      });
```

Comment:
There is a typo in the `exports.purchaseHistory` function. It should be `res.json(orders)` instead of `res.jsosn(orders)`.